demo-java/app/src/main/res/values/strings.xml

```diff
@@ -140,7 +140,7 @@
 
   <!-- Text for the find current place submit button. -->
   <string name="find_current_place_button" translatable="false">Find Current Place</string>
-  <string name="programmatic_autocomplete">Programmatic Autocomplete</string>
+  <string name="programmatic_as_you_type_autocomplete">Programmatic As-You-Type Autocomplete</string>
```

Comment:
I would rename this "Programmatic Autocomplete + Geocoding" since "as you type" is expected, and it's the addition of Geocoding that is distinct about this implementation.